tyr-runner/src/main/java/org/jboss/tyr/ci/ContinuousIntegration.java

```diff
@@ -19,9 +19,9 @@
 
 public interface ContinuousIntegration {
 
-    void triggerBuild(JsonObject prPayload);
+    void triggerRetesting(JsonObject prPayload);
```

Comment:
what is wrong with the original?

Reply:
There still is the original method https://github.com/khermano/tyr/commit/0bbcf1a3ad81bba448f8b38d878b73449c539915#diff-2495a7c9254abaaeb22b96823ce7bb5aR103 .... I just want to separate its functionality because of DRY principle as I use this method in both triggerRetesting() and triggerRetestingFailed() methods.  :)

Reply:
that is a private method in the implementation. That's not a reason to rename interface.

Reply:
Sure, I understand your point. I will rename it back and change name of the private method. :)